test/common.go

```diff
@@ -20,10 +22,14 @@ var arrayUsername = flag.String("arrayUsername", "xxx", "Array Username")
 
 var arrayPassword = flag.String("arrayPassword", "xxx", "Array Password")
 
+var downstreamArrayIP = flag.String("downstream", "", "Array IP to be used as downstream")
+
 // Required for group merge test
-const sourceArrayIP = "1.1.1.2"
-const sourceArrayusername = "xxx"
-const sourceArraypassword = "xxx"
+var sourceArrayIP = flag.String("sourceArrayIP", "1.1.1.2", "Source array IP for group tests")
+
+var sourceArrayusername = flag.String("sourceArrayusername", "xxx", "Source array username")
```

Comment:
The flag names are getting long.  It's not a problem, but it might be inconvenient.

Also, the 'u' in 'username' should be upper case.

Reply:
I couldnt change this value since Kousalya has a PR out which uses lower case username.
@kousalya-hpe can you please update this change in your PR too?